scripts/linters/pylint_extensions_test.py

```diff
@@ -3272,3 +3272,96 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class StringConcatenationCheckerTests(unittest.TestCase):
```

Comment:
One more small suggestion - Please include one test to check whether the error msg string is being correctly generated or not - i.e the generated error string is exactly equal to what we expect or not. (We are slowly trying to enforce this practice in our codebase)